### PR TITLE
Resolve TypeError when clearing new changes in articleform

### LINKED ISSUE #4906: Editor: TypeError when clearing new changes
**Describe the bug**
```
Uncaught TypeError: Cannot read property 'id' of undefined
    at Object.value [as click] (articleForm.jsx:311)
    at HTMLButtonElement.v (preact.mjs:183)
```

**To Reproduce**

1. Edit a article
2. Make some changes
3. click on clear on bottom, right of "new changes"
4. The "Are you sure you want to revert to the previous save?" popup appears
5. click yes
6. Changes are not cleared as expected. In the console the above error is logged.

**Expected behavior**

The new changes to be cleared
**Screenshots**

**Desktop (please complete the following information):**

- OS: win10
- Browser: chrome
- Version: Version 78.0.3904.108 (Official Build) (64-bit)

**Smartphone (please complete the following information):**

N/A on laptop

- Device:
- OS:
- Browser:
- Version:

**Additional context**
I tried reloading the page and consistently ran into the error.

### PR DESCRIPTION
**Pull request by bolariin**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Documentation Update

## Description
It looks like the error was caused by a typo in `articleForm.jsx`.

## Related Tickets & Documents
Resolves #4906

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
clear-changes-working-as-expected

## Added to documentation?
- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

### CODE REVIEW

File: app/javascript/article-form/articleForm.jsx [hunk 1]
```diff
@@ -308,7 +308,7 @@ export default class ArticleForm extends Component {
       previewResponse: '',
       helpHTML: document.getElementById('editor-help-guide').innerHTML,
       submitting: false,
-      editing: this.artical.id !== null, // eslint-disable-line react/no-unused-state
+      editing: this.article.id !== null, // eslint-disable-line react/no-unused-state
```
Comment (nickytonline) [inline app/javascript/article-form/articleForm.jsx]: 👍

File: app/javascript/article-form/elements/publishToggle.jsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ const PublishToggle = ({
       </button>
     )}
     <span>
-      <p style={!edited && { visibility: 'hidden' }}>
+      <p style={!edited && { visibility: 'hidden', cursor: 'pointer' }}>
```
Comment (nickytonline) [inline app/javascript/article-form/elements/publishToggle.jsx]: I'm not following why we need to change the cursor style on a hidden paragraph.

Comment (bolariin) [reply app/javascript/article-form/elements/publishToggle.jsx]: Oh, I thought it changed the cursor on the paragraph when it becomes visible

Comment (bolariin) [reply app/javascript/article-form/elements/publishToggle.jsx]: I definitely made a mistake here, I thought the cursor didn't change but trying it on master again everything looks fine.

**Comment by Almenon** [comment]
That worked, thanks @bolariin !

**Pull request merged**